src/components/NavBar/NavBar.vue

```diff
@@ -66,6 +71,12 @@ export default {
         this.$router.push({ name: "Login" });
       });
     },
+    redirect(symbol){
+      const regex = /^\S+$/;
```

Comment:
🎯

Reply:
I do believe it was written in the requirements this way. For the MVP anyways.

Reply:
yes, it is what the product owners asked for and what we agreed to do.